src/cpp/rtps/transport/UDPTransportInterface.cpp

```diff
@@ -61,6 +61,8 @@ UDPTransportInterface::UDPTransportInterface(
     : TransportInterface(transport_kind)
     , mSendBufferSize(0)
     , mReceiveBufferSize(0)
+    , first_time_open_output_channel_(true)
+    , whitelisted_(false)
```

Comment:
After adding it to UDPSenderResource, it seems this whitelisted_ flag is not used.

```suggestion
    , first_time_open_output_channel_(true)
```

Reply:
Good catch. It was added because it was required for a previous solution. Now that the `AnyAddressEndpoint` is out bounded to the localhost interface, is no longer necessary.